src/s3PackageManager.js

```diff
@@ -22,7 +22,7 @@ export default class S3PackageManager implements ILocalPackageManager {
     this.config = config;
     this.packageName = packageName;
     this.logger = logger;
-    this.s3 = new S3({ endpoint: config.endpoint, region: config.region });
+    this.s3 = new S3({ endpoint: this.config.endpoint, region: this.config.region, s3ForcePathStyle: this.config.s3ForcePathStyle });
```

Comment:
Now that this is longer, can you split in across multiple lines for easier future diffing? (both here and in index.js)

```js
this.s3 = new S3({
    endpoint: this.config.endpoint,
    region: this.config.region,
    s3ForcePathStyle: this.config.s3ForcePathStyle
});
```

Reply:
I formatted the line in both places and squashed the commit!